chainweb.cabal

```diff
@@ -712,7 +724,17 @@ executable transaction-generator
         , servant >= 0.14.1
         , servant-client >= 0.14
         , servant-client-core >= 0.14
+        , stm
+        , stm-chans
         , text >= 1.2
+        , time >= 1.8
+        , unordered-containers >= 0.2
+        , wai >= 3.2
+        , wai-app-static >= 3.1
+        , wai-extra >= 3.0
+        , wai-cors >= 0.2
+        , warp >= 3.2
```

Comment:
Why are we depending on warp and all these wai packages?  The transaction generator shouldn't need to run a web server.

Reply:
I needed the `Utils.Logging`  and `Utils.Logging.Config` modules from the `node` folder. Requiring those modules pulled in those extra dependencies, unfortunately.

Reply:
Generally you only need the deps if your code directly, and for instance you are not directly issuing CORS configurations so I'm sure you don't need `wai-cors` declared here.

Reply:
I've addressed this issue in a new commit.